V/VARNA/build_tarballs.jl

```diff
@@ -0,0 +1,34 @@
+using BinaryBuilder, Pkg
+
+name = "VARNA"
+version = v"3.93.0"
+
+# url = "https://varna.lisn.upsaclay.fr/"
+# description = "Java component and applet for drawing RNA secondary structure"
+
+sources = [
+    FileSource("https://varna.lisn.upsaclay.fr/bin/VARNAv$(version.major)-$(version.minor).jar",
+               "276996b0eb54ab7bdbf92a113f0c23d8d3ed8e497d85b6577b3f2065a0dbcf87"),
+]
+
+script = raw"""
+cd $WORKSPACE/srcdir/
+
+install -Dvm 644 VARNA*.jar ${prefix}/share/VARNA.jar
+
+# VARNA uses GPL as stated here: https://varna.lisn.upsaclay.fr/
+install_license /usr/share/licenses/GPL-3.0+
+"""
+
+platforms = supported_platforms()
```

Comment:
If anything, this should be `[AnyPlatform()]`, but I'm missing the point of this being a JLL

Reply:
Ok I am using an artifact now. Wishing the artifacts system were as easy to use as BinaryBuilder.